source/appModules/explorer.py

```diff
@@ -221,10 +233,13 @@ def chooseNVDAObjectOverlayClasses(self, obj, clsList):
 				clsList.insert(0, ImmersiveLauncher)
 			elif uiaClassName=="ListViewItem" and obj.UIAElement.cachedAutomationId.startswith('Suggestion_'):
 				clsList.insert(0,SuggestionListItem)
-			elif uiaClassName=="MultitaskingViewFrame" and role==controlTypes.ROLE_WINDOW:
-				clsList.insert(0,MultitaskingViewFrameWindow)
-			elif obj.windowClassName=="MultitaskingViewFrame" and role==controlTypes.ROLE_LISTITEM:
-				clsList.insert(0,MultitaskingViewFrameListItem)
+			elif uiaClassName=="MultitaskingViewFrame":

```

Comment:
@feerrenrut @leonardder While this seems logical I'd like your expert opinion.

Reply:
I'm happy with this as long as you manually confirm that `uiaClassName` is equal to `obj.windowClassName` when looking at the "MultitaskingViewFrame"

Reply:
I dug a bit deeper in the code and reverted this change. However, while reading the function I also tried to improve readability and consistency of code style a bit.

Reply:
If you found out why, could you add a comment here about why this one uses `windowClassName` rather than `uiaClassName`?